api/lib/opentelemetry.rb

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 OpenTelemetry Authors
```

Comment:
```suggestion
# Copyright 2019 OpenTelemetry Authors
# SPDX-License-Identifier: Apache-2.0
```

Might be more convenient to use `SPDX-License-Identifier` header + dedicated `LICENSE` file in repo root, instead of verbose header in each file. Thoughts?

Example: https://github.com/kubernetes-sigs/kustomize/blob/master/cmd/kustomize/main.go#L1